envsec/pkg/envcli/flags.go

```diff
@@ -105,14 +107,25 @@ func (f *configFlags) genConfig(cmd *cobra.Command) (*CmdConfig, error) {
 		}
 	}
 
-	ssmConfig, err := awsfed.GenSSMConfigFromToken(ctx, tok, true /*useCache*/)
-	if err != nil {
-		return nil, errors.WithStack(err)
+	var store envsec.Store
+	if !build.IsDev {
```

Comment:
This is ok in a pinch, but we probably want to add a env var that determines which backend is used

Reply:
Yeah, I think we need some design thought into how to specify the backend for the envsec project. 

It may be part of `envsec init`, and then stored in .... an `envsec.json`? 

I may be missing something but I don't see how env-var would be ideal, because the users of a particular envsec-project should use the same backend store (i.e. all members of a team should use the same backend store for their project).